include/drivers/pwm.h

```diff
@@ -60,6 +62,12 @@ extern "C" {
 /** PWM pin capture captures period/pulse width continuously. */
 #define PWM_CAPTURE_MODE_CONTINUOUS	(1U << PWM_CAPTURE_MODE_SHIFT)
 
+/**
+ * PWM pin force update applies pin configuration immediately, this
+ * may reset the timer counter register.
```

Comment:
I'd change flag description to:
```suggestion
 * PWM pin configuration is applied immediately instead of waiting until next period.
```